pkg/provider/provider.go

```diff
@@ -3,23 +3,26 @@ package provider
 import (
 	"context"
 	"fmt"
-
 	"github.com/EnterpriseDB/terraform-provider-biganimal/pkg/api"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/provider"
+	schema2 "github.com/hashicorp/terraform-plugin-framework/provider/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	diagv2 "github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"os"
```

Comment:
`"os"` should go to the beginning of the imports. I guess this is the difference between goimports and gofmt.